koopovereenkomst-v3-simple/pages/_document.tsx

```diff
@@ -0,0 +1,89 @@
+import { EmotionJSX } from "@emotion/react/types/jsx-namespace";
+import createEmotionServer from "@emotion/server/create-instance";
+import Document, {
+  DocumentContext, DocumentProps, Head, Html, Main,
+  NextScript
+} from "next/document";
+import createEmotionCache from "../src/createEmotionCache";
+import { theme, roboto } from "../src/theme";
+
+interface MyDocumentProps extends DocumentProps {
+  emotionStyleTags: EmotionJSX.Element[];
+}
+
+export default function MyDocument({ emotionStyleTags }: MyDocumentProps) {
+  return (
+    <Html lang="en" className={roboto.className}>
```

Comment:
Hebben we de `index.html`  ook verwijderd? (kan ik niet zo snel terug vinden ... ) 🤔

Reply:
Yep, kan 'm zelf ook niet vinden 👍